### PR TITLE
Adds aws_s3_bucket_public_access_block for file_storage bucket

### PR DESCRIPTION
**Pull request by zythosec**
This will ensure even if https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_account_public_access_block is not specified in a customer account that this bucket cannot have public objects or public policies attached.

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_public_access_block
https://docs.aws.amazon.com/AmazonS3/latest/userguide/access-control-block-public-access.html

### CODE REVIEW

File: modules/file_storage/main.tf [hunk 1]
```diff
@@ -54,6 +54,12 @@ resource "aws_s3_bucket" "file_storage" {
   depends_on = [aws_sqs_queue.file_storage]
 }
 
+resource "aws_s3_bucket_public_access_block" "example" {
```
Comment (jsbroks) [inline modules/file_storage/main.tf:57]: probably not call it example

**Review by jsbroks — CHANGES_REQUESTED** [review]

File: modules/file_storage/main.tf (hunk 1 above)
Comment (zythosec) [reply modules/file_storage/main.tf:57]: 🤦🏼

File: modules/file_storage/main.tf [hunk 2]
```diff
@@ -54,6 +54,12 @@ resource "aws_s3_bucket" "file_storage" {
...
+	bucket = aws_s3_bucket.file_storage.id
+	block_public_acls   = true
+	block_public_policy = true
+}
```
[the remark this answers is not in the document]
Comment (zythosec) [reply modules/file_storage/main.tf:61]: Oh nice didn't know this was hooked up.

**Review by jsbroks — APPROVED** [review]

**Comment by jsbroks** [comment]
This PR is included in version 1.0.0 :tada:

**Pull request merged**